src/ridgeplot/_color/interpolation.py

```diff
@@ -115,13 +115,17 @@ def _mul(a: tuple[Numeric, ...], b: tuple[Numeric, ...]) -> tuple[Numeric, ...]:
 
 
 def _interpolate_row_index(ctx: InterpolationContext) -> ColorscaleInterpolants:
+    if ctx.n_rows <= 1:
+        return [[0.0] * len(row) for row in ctx.densities]
     return [
         [((ctx.n_rows - 1) - ith_row) / (ctx.n_rows - 1)] * len(row)
         for ith_row, row in enumerate(ctx.densities)
     ]
 
 
 def _interpolate_trace_index(ctx: InterpolationContext) -> ColorscaleInterpolants:
+    if ctx.n_traces <= 1:
+        return [[0.0] * len(row) for row in ctx.densities]
```

Comment:
Small simplification:
```suggestion
        return [[0.0]]
```

Reply:
I see you added this to all cases. Unfortunately, this simplification only applies to this specific case (`_interpolate_trace_index`). Let me revert it...